antora-playbook.yaml

```diff
@@ -0,0 +1,141 @@
+          .doc h5 { font-size: 1.1rem; font-weight: 500; text-decoration: underline; }
+          .doc h6 { font-size: 0.9rem; font-weight: 500; text-decoration: underline; }
+          /* Default `code`, `pre`, and `.colist` (source code annotations) fonts are too big, adjust them: */
+          .doc .colist>table code, .doc p code, .doc thead code { font-size: 0.8em; }
+          .doc pre { font-size: 0.7rem; }
+          .doc .colist { font-size: 0.75rem; }
+          /* Make links more visible: */
+          .doc a { text-decoration: underline; }
+          .doc a code { text-decoration: underline; color: #1565c0; }
+          /* Tab header fonts aren't rendered good, adjusting the font weight: */
+          .tablist > ul li { font-weight: 500; }
+          /* `page-toclevels` greater than 4 are not supported by Antora UI, patching it: */
+          .toc .toc-menu li[data-level="4"] a {
+            padding-left: 2.75rem;
+          }
+          /* Replace the default highlight.js color for strings from red (unnecessarily signaling something negative) to green: */
+          .hljs-string {
+            color: #0f8532;
+          }
+        </style>
+
+    - path: partials/header-content.hbs
+      contents: |
+        <header class="header">
+          <nav class="navbar">
+            <div class="navbar-brand">
+              <span class="navbar-item title">{{site.title}}</span>
+            </div>
+            <div id="topbar-nav" class="navbar-menu">
+              <div class="navbar-end">
+                <a class="navbar-item" href="https://logging.apache.org">a subproject of&nbsp;<strong>Apache Logging Services</strong></a>
+              </div>
+            </div>
+          </nav>
+        </header>
+
+    - path: partials/footer-content.hbs
+      contents: |
+        <footer class="footer">
+          <p>
+            Copyright © 1999-{{{year}}} <a href="https://www.apache.org/">The Apache Software Foundation</a>.
+            Licensed under the <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache Software License, Version 2.0</a>.
+            Please read our <a href="https://privacy.apache.org/policies/privacy-policy-public.html">privacy policy</a>.
+          </p>
+          <p>
+            Apache, Log4j, and the Apache feather logo are trademarks or registered trademarks of The Apache Software Foundation.
+            Oracle and Java are registered trademarks of Oracle and/or its affiliates.
+            Other names may be trademarks of their respective owners.
+          </p>
+        </footer>
+
+    # Disable component version selector
+    - path: partials/nav-explore.hbs
+      contents: ""
+
+    # Fix the `Edit this page` link
+    - path: partials/edit-this-page.hbs
+      contents: |
+        <div class="edit-this-page"><a href="https://github.com/apache/logging-log4net/edit/Feature/Antora/src/site/antora/modules/{{page.module}}/pages/{{page.relativeSrcPath}}">Edit this Page</a></div>
```

Comment:
@FreeAndNil, I presume you fixed this after the merge?

Reply:
Yes, I already have fixed this.